rally_ovs/plugins/ovs/ovnclients.py

```diff
@@ -44,6 +44,10 @@ def _restart_daemon(self):
         self._stop_daemon()
         return self._start_daemon()
 
+    def _get_gw_ip(self, network_cidr):
```

Comment:
@hzhou8 if we use the first IP to create the LRP then scenarios where `lswitch_create_args.start_cidr == "42.0.0.0/16"` would generate LRPs with IP 42.0.0.0. Some network stacks might not like that, e.g. linux `ping` considers that a broadcast IP. This will break ping tests from simulated VMs to the default gateway.

If we use the last non broadcast IP in the range as LRP address then we're fine. We'd be creating LRP with IP 42.0.254.253 and simulated VMs with IPs starting from 42.0.0.1.

On the other hand, we could say that it's invalid to configure `start_cidr == "42.0.0.0/16"` in which case we could use the old allocation scheme. But afaik we don't do such checks anywhere in the code.

I'm ok to switch back to the previous behavior if you prefer that one more.

Reply:
> 
> On the other hand, we could say that it's invalid to configure `start_cidr == "42.0.0.0/16"` in which case we could use the old allocation scheme. But afaik we don't do such checks anywhere in the code.
> 
> I'm ok to switch back to the previous behavior if you prefer that one more.

@hzhou8 it's also worth to note that now with the code on master branch we would overwrite any `lswitch_create_args.start_cidr == "42.0.0.1/16"` with `lswitch_create_args.start_cidr == "42.0.0.0/16"` in `create_routed_network()`:
https://github.com/ovn-org/ovn-scale-test/blob/393bd0ff1f40e0bb911a711110a5671f100e78aa/rally_ovs/plugins/ovs/scenarios/ovn_nb.py#L44